src/main/resources/TotemCode/Translations.en.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This change is not complete as we are missing the other translations. It could be also a separate commit for the translations. The english translations should be in the default Translation page with a defaultLanguage set to "en". The french translation should be in the french page.